### PR TITLE
Add first-timers-only-app to website

### PR DESCRIPTION
**Pull request by Akshitaag**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: {explain the action here}
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](yourURLhere)
- [x] [Has documentation](yourURLhere)
- [x] [Is open source](yourURLhere)
- [x] [Has a license](yourURLhere)
- [x] [Has a code of conduct](yourURLhere)
- [x] Has someone willing to at least minimally maintain them for the near future: {your maintainer's name}

### CODE REVIEW

**Comment by Akshitaag** [comment]
@hiimbex I made the changes! Let me know if there is still any error.

File: _apps/first-timers-only-app.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: First-timers-only-app
+description: A bot that ensures exactly one newcomer issue is assigned per person so that it’s fair for all newcomers.
+flug: ftoa
+screenshots:
+- https://github.com/probot/first-timers-only-app/blob/master/Snapshots/readmeSS.png
+authors:
+-akshitaag
+-rahmeen14
+repository: https://github.com/probot/first-timers-only-app
+host: https://sneaky-glass.glitch.me/probot
```
Comment (hiimbex) [inline _apps/first-timers-only-app.md]: `https://sneaky-glass.glitch.me/probot` => `https://sneaky-glass.glitch.me`

File: _apps/first-timers-only-app.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: First-timers-only-app
+description: A bot that ensures exactly one newcomer issue is assigned per person so that it’s fair for all newcomers.
+flug: ftoa
+screenshots:
+- https://github.com/probot/first-timers-only-app/blob/master/Snapshots/readmeSS.png
+authors:
+-akshitaag
+-rahmeen14
```
Comment (hiimbex) [inline _apps/first-timers-only-app.md]: I think you'll need spaces here ie:
```
authors:
- akshitaag
- rahmeen14
```

File: _apps/first-timers-only-app.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: First-timers-only-app
+description: A bot that ensures exactly one newcomer issue is assigned per person so that it’s fair for all newcomers.
```
Comment (hiimbex) [inline _apps/first-timers-only-app.md]: What about:
> Ensures first timer issues are assigned to exactly one first timer

File: _apps/first-timers-only-app.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+screenshots:
+- https://github.com/probot/first-timers-only-app/blob/master/Snapshots/readmeSS.png
+authors:
+-akshitaag
+-rahmeen14
+repository: https://github.com/probot/first-timers-only-app
+host: https://sneaky-glass.glitch.me/probot
+-------
+
+Statement:
```
Comment (hiimbex) [inline _apps/first-timers-only-app.md]: You can still use markdown here!

**Pull request closed without merging**